tests/award.js

```diff
@@ -377,7 +376,6 @@ module.exports = {
                             .end();
                         done();
                     })
-                done();
```

Comment:
there's an issue with this test - if you look there's 2 nested .perform methods being called and they both have a "done" param. So the inner done is shadowing the outer done which leads to some weird behavior. Plus, I think all the async stuff in .perform methods isn't even completing before the test finishes.